frontend/src/home/Home.tsx

```diff
@@ -222,142 +237,39 @@ class HomeComponent extends React.Component<Props> {
-    );
-
-    if (plan) {
-      this.props.setCoopCycle(plan);
-    }
-  }
-
-  renderMajorDropDown() {
-    return (
-      <Autocomplete
-        style={{ width: 300, marginRight: 18 }}
-        disableListWrap
-        options={this.props.majors.map(maj => maj.name)}
-        renderInput={params => (
-          <TextField
-            {...params}
-            variant="outlined"
-            label="Select A Major"
-            fullWidth
-          />
-        )}
-        value={!!this.props.major ? this.props.major.name + " " : ""}
-        onChange={this.onChooseMajor.bind(this)}
-      />
-    );
-  }
-
-  renderPlansDropDown() {
-    return (
-      <Autocomplete
-        style={{ width: 300 }}
-        disableListWrap
-        options={[
-          "None",
-          ...this.props.plans[this.props.major!.name].map(p => planToString(p)),
-        ]}
-        renderInput={params => (
-          <TextField
-            {...params}
-            variant="outlined"
-            label="Select A Plan"
-            fullWidth
-          />
-        )}
-        value={this.props.planStr || "None"}
-        onChange={this.onChoosePlan.bind(this)}
-      />
-    );
-  }
-
-  renderSetClassesButton() {
-    return (
-      <Button
-        variant="contained"
-        color="secondary"
-        style={{ marginLeft: 18 }}
-        onClick={() => this.addClassesFromPOS()}
-      >
-        Set schedule to example from plan of study
-      </Button>
-    );
-  }
-
-  addClassesFromPOS() {
-    const plan = this.props.plans[this.props.major!.name].find(
-      (p: Schedule) => planToString(p) === this.props.planStr!
-    );
-    this.props.setSchedule(plan!);
-  }
-
-  renderClearScheduleButton() {
-    return (
-      <Button
-        variant="contained"
-        color="secondary"
-        style={{ marginLeft: 18 }}
-        onClick={() => this.clearSchedule()}
-      >
-        Clear Schedule
-      </Button>
-    );
-  }
-
-  clearSchedule() {
-    const plan = this.props.plans[this.props.major!.name].find(
-      (p: Schedule) => planToString(p) === this.props.planStr!
-    );
-    this.props.setCoopCycle(plan!);
-  }
-
   renderYears() {
     return this.props.schedule.years.map((year: number, index: number) => (
       <Year key={index} index={index} schedule={this.props.schedule} />
     ));
   }
 
   render() {
-    const { schedule, major, planStr } = this.props;
     return (
       <OuterContainer>
         <DragDropContext
           onDragEnd={this.onDragEnd}
           onDragUpdate={this.onDragUpdate}
         >
           <Container>
-            <div /* onClick={() => console.log(this.state)} */>
+            <HomeTop>
+              <HomeText href="#">GraduateNU</HomeText>
+              <HomePlan>
+                <MajorText>
+                  {!!this.props.major ? this.props.major.name + ": " : ""}
+                </MajorText>
+                <PlanText>{this.props.planStr || "None"}</PlanText>
+                <EditPlanPopper />
```

Comment:
I believe that there is a bunch of code, props, that are redundant in the HomeScreen given that a lot of the  Major/Plan switching logic got move to the EditPlanPopper. You  can get rid of the `setSchedule`, `setCoopCycle` `setMajor` from `ReduxDispatchHomeProps` and the associated callback assignments in `mapDispatchToProps` . You can also get rid of the `major`, `majors`, `plans` properties of the `ReduxStoreHomeProps` as well as associated callback assignments in  `mapStateToProps`